payments-core/src/main/java/com/stripe/android/payments/financialconnections/FinancialConnectionsAvailability.kt

```diff
@@ -0,0 +1,9 @@
+package com.stripe.android.payments.financialconnections
+
+import androidx.annotation.RestrictTo
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+enum class FinancialConnectionsAvailability(val available: Boolean) {
```

Comment:
Nit: We can get rid of `available` now.

Reply:
https://github.com/stripe/stripe-android/pull/10504/commits/788dda569279d2997c45d54bd968871d5dbb0d1e